src/pages/Vlrt.jsx

```diff
@@ -1,61 +1,147 @@
 /* eslint-disable no-param-reassign */
 /* eslint-disable react/no-array-index-key */
-import React, { useEffect, useState } from "react";
+import React, { useState } from "react";
 import PlayerVlrt from "components/PlayerVlrt.jsx";
 
 const players = Array.from({ length: 10 }, (_, index) => `Player ${index + 1}`);
 export default function Vlrt() {
-  const [playerData, setPlayerData] = useState([]);
-
-  useEffect(() => {
-    const tierColors = {
-      Iron: "#5E5E5E",
-      Bronze: "#8D6F46",
-      Silver: "#BAC8D1",
-      Gold: "#FFD700",
-      Platinum: "#54D3E0",
-      Diamond: "#A770F0",
-      Ascendant: "#1CB952",
-      Immortal: "#C23A73",
-      Radiant: "#F3992E",
-    };
-
-    const labels = document.querySelectorAll(".players__tiers");
-
-    labels.forEach((label) => {
-      const tierText = label.textContent.trim();
-      if (tierColors[tierText]) {
-        label.style.color = tierColors[tierText];
+  const [playerData, setPlayerData] = useState(
+    players.map((_, index) => ({
+      playerName: `Player ${index + 1}`,
+      tier: "Iron",
+      pts: 7,
+    }))
+  );
+
+  const tierToPoints = {
+    Iron: 7,
+    Bronze: 13,
+    Silver: 17,
+    Gold: 25,
+    Platinum: 29,
+    Diamond: 37,
+    Ascendant: 43,
+    Immortal: 45,
+    Radiant: 48,
+  };
+
+  const handlePlayerChange = (index, field, value) => {
```

Comment:
이런 함수가 있을 때, 매개변수를 객체로 받는 방법이 좋습니다.
가독성도 향상되고, 향후 새로운 변수가 추가될 때 추가하기가 쉽구요. 파라미터 순서를 기억할 필요도 없을거예요.

```suggestion
const handlePlayerChange = ({ index, field, value }) => {
```

Reply:
추가적으로 타입스크립트를 사용하기 전이니, 그 전까지는 함수를 생성했을 때 설명이 필요한 경우 아래를 참고해서 JSDoc 을 추가하면 좋아요.

https://jsdoc.app/